modules/apps/export-import/export-import-test/src/testIntegration/java/com/liferay/exportimport/internal/content/processor/test/DefaultExportImportContentProcessorTest.java

```diff
@@ -239,8 +240,15 @@ public void testExportDLReferences() throws Exception {
 		String content = _replaceParameters(
 			_getContent("dl_references.txt"), _fileEntry);
 
-		_exportImportContentProcessor.validateContentReferences(
-			_stagingGroup.getGroupId(), content);
+		try {
+			_exportImportContentProcessor.validateContentReferences(
+				_stagingGroup.getGroupId(), content);
+		}
+		catch (Exception exception) {
+			if (exception.getCause() instanceof NoSuchFileEntryException) {
+				Assert.fail("Filename should not be recognised as UUID");
+			}
+		}
```

Comment:
Shouldn't you use `@Test(expected = ...)` instead of `Assert.fail`?

Reply:
I figured manually failing the test on a given exception is acceptable, as once I recieve the exception I expect the test to fail

Reply:
I think it would be better to create a new (and smaller) test that check this specific condition. This test is quite big and hairy already.

Can you create a new `testExportInvalidUUIDDLReference` test method, check this particular case there and use `@Test(expected = ...)`?

Reply:
Sure I'll create a new test for it

Reply:
I abstracted the try catch into a new test, and I asserted the exception type to warn the developer about the uuid. In case the fix is reverted other tests will fail as well, but with the exception and won't be that verbose.